### ISSUE TITLE
Animepahe no longer working

### ISSUE THREAD

**Issue by trunc88** [justfoolingaround/animdl#131]
**Describe the bug**
Trying to stream/download from Animepahe does not work anymore.
```
$ animdl stream "animepahe:dragon ball" -r 56
                                                                               justfoolingaround/animdl - v1.5.4
                                                                        A highly efficient anime downloader and streamer
Traceback (most recent call last):
  File "/home/xxxx/.local/bin/animdl", line 8, in <module>
    sys.exit(__animdl_cli__())
  File "/home/xxxx/.local/lib/python3.10/site-packages/click/core.py", line 1128, in __call__
    return self.main(*args, **kwargs)
  File "/home/xxxx/.local/lib/python3.10/site-packages/click/core.py", line 1053, in main
    rv = self.invoke(ctx)
  File "/home/xxxx/.local/lib/python3.10/site-packages/click/core.py", line 1659, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/xxxx/.local/lib/python3.10/site-packages/click/core.py", line 1395, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/xxxx/.local/lib/python3.10/site-packages/click/core.py", line 754, in invoke
    return __callback(*args, **kwargs)
  File "/home/xxxx/.local/lib/python3.10/site-packages/animdl/core/cli/helpers/fun.py", line 135, in internal
    return f(*args, **kwargs)
  File "/home/xxxx/.local/lib/python3.10/site-packages/animdl/core/cli/commands/stream.py", line 142, in animdl_stream
    anime, provider = helpers.process_query(
  File "/home/xxxx/.local/lib/python3.10/site-packages/animdl/core/cli/helpers/processors.py", line 67, in process_query
    return prompt_user(logger, genexp, searcher.provider)
  File "/home/xxxx/.local/lib/python3.10/site-packages/animdl/core/cli/helpers/processors.py", line 9, in prompt_user
    expansion = [*anime_list_genexp]
  File "/home/xxxx/.local/lib/python3.10/site-packages/animdl/core/cli/helpers/searcher.py", line 85, in search_animepahe
    content = animepahe_results.json()
  File "/home/xxxx/.local/lib/python3.10/site-packages/httpx/_models.py", line 1517, in json
    return jsonlib.loads(self.text, **kwargs)
  File "/usr/lib/python3.10/json/__init__.py", line 346, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python3.10/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python3.10/json/decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
```
`animdl` 1.5.4, also tested many versions back to 1.4.83, and a fresh pipx install
`python` 3.10.4
linux

**Did the error occur in between an active task (`download` / `stream` / `grab`)?**
The same error occurs when trying to `download`, `stream`, or `grab`. Not quite sure what you mean by "in between", sorry.

**Comment by justfoolingaround** [comment]
Fixed as of `1.5.75`.

**Comment by trunc88** [comment]
Thank you!
